doc/classes/String.xml

```diff
@@ -338,193 +377,207 @@
 		</method>
 		<method name="lpad" qualifiers="const">
 			<return type="String" />
 			<param index="0" name="min_length" type="int" />
 			<param index="1" name="character" type="String" default="&quot; &quot;" />
 			<description>
-				Formats a string to be at least [param min_length] long by adding [param character]s to the left of the string.
+				Formats the string to be at least [param min_length] long by adding [param character]s to the left of the string, if necessary. See also [method rpad].
 			</description>
 		</method>
 		<method name="lstrip" qualifiers="const">
 			<return type="String" />
 			<param index="0" name="chars" type="String" />
 			<description>
-				Returns a copy of the string with characters removed from the left. The [param chars] argument is a string specifying the set of characters to be removed.
-				[b]Note:[/b] The [param chars] is not a prefix. See [method trim_prefix] method that will remove a single prefix string rather than a set of characters.
+				Removes a set of characters defined in [param chars] from the string's beginning. See also [method rstrip].
+				[b]Note:[/b] [param chars] is not a prefix. Use [method trim_prefix] to remove a single prefix, rather than a set of characters.
 			</description>
 		</method>
 		<method name="match" qualifiers="const">
 			<return type="bool" />
 			<param index="0" name="expr" type="String" />
 			<description>
-				Does a simple case-sensitive expression match, where [code]"*"[/code] matches zero or more arbitrary characters and [code]"?"[/code] matches any single character except a period ([code]"."[/code]). An empty string or empty expression always evaluates to [code]false[/code].
+				Does a simple expression match, where [code]*[/code] matches zero or more arbitrary characters and [code]?[/code] matches any single character except a period ([code].[/code]). An empty string or empty expression always evaluates to [code]false[/code].
 			</description>
 		</method>
 		<method name="matchn" qualifiers="const">
 			<return type="bool" />
 			<param index="0" name="expr" type="String" />
 			<description>
-				Does a simple case-insensitive expression match, where [code]"*"[/code] matches zero or more arbitrary characters and [code]"?"[/code] matches any single character except a period ([code]"."[/code]). An empty string or empty expression always evaluates to [code]false[/code].
+				Does a simple [b]case-insensitive[/b] expression match, where [code]*[/code] matches zero or more arbitrary characters and [code]?[/code] matches any single character except a period ([code].[/code]). An empty string or empty expression always evaluates to [code]false[/code].
 			</description>
 		</method>
 		<method name="md5_buffer" qualifiers="const">
 			<return type="PackedByteArray" />
 			<description>
-				Returns the MD5 hash of the string as an array of bytes.
+				Returns the [url=https://en.wikipedia.org/wiki/MD5]MD5 hash[/url] of the string, as a [PackedByteArray].
```

Comment:
There shouldn't be a comma before "as", as when you add a comma, its meaning changes to "because". This isn't the only method with this mistake.